src/components/common/icon-button/icon-button.styles.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+
+// =============================================================================
+// STYLING INTERFACE
+// =============================================================================
+interface IconButtonStyleProps {
```

Comment:
prefix with `I` for interfaces

```suggestion
interface IIconButtonStyleProps {
```